drivers/gles3/shaders/scene.glsl

```diff
@@ -2144,6 +2140,14 @@ FRAGMENT_SHADER_CODE
 		diffuse_light *= rev_amount;
 	}
 
+
+#ifdef DIFFUSE_TOON
+
+    diffuse_light *= step(0.1, max(diffuse_light.r, max(diffuse_light.g, diffuse_light.b)));
+
+
+# endif // DIFFUSE_TOON
```

Comment:
Style issues (indentation should be tabs, and two empty lines of separation is too much):
```suggestion
#ifdef DIFFUSE_TOON

	diffuse_light *= step(0.1, max(diffuse_light.r, max(diffuse_light.g, diffuse_light.b)));

# endif // DIFFUSE_TOON
```